src/openforms/js/utils/translation.js

```diff
@@ -1,4 +1,8 @@
+import FormioUtils from 'formiojs/utils';
```

Comment:
The form
```javascript
import {Utils as FormioUtils} from 'formiojs';
```

trips jest with `SyntaxError: Unexpected token 'export'`. This phrasing doesn't. [Other workarounds exist](https://www.sobyte.net/post/2022-06/jest/)